test/index.js

```diff
@@ -1163,6 +1163,36 @@ describe('either', function() {
 
   });
 
+  describe('maybeToEither', function() {
+
+    it('is a binary function', function() {
+      eq(typeof S.either, 'function');
```

Comment:
I spotted this typo yesterday. Fixed in 72da0c9693c32774838aea6bcb68e48414ef2fb7. :)